packages/legend-studio/style/components/editor/_external-format-editor.scss

```diff
@@ -367,14 +367,19 @@
     background: var(--color-dark-grey-50);
     color: var(--color-light-grey-200);
 
-    &__active {
+    &__active,
```

Comment:
I don't think you need to style these at all if we use `menu` properly. Can you remove these `context-menu` classes and styles and try?